src/main/java/org/openstreetmap/atlas/geography/atlas/multi/SubEdgeList.java

```diff
@@ -0,0 +1,50 @@
+package org.openstreetmap.atlas.geography.atlas.multi;
+
+import java.io.Serializable;
+import java.util.Iterator;
+import java.util.List;
+
+import org.openstreetmap.atlas.exception.CoreException;
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+import org.openstreetmap.atlas.geography.atlas.items.Edge;
+import org.openstreetmap.atlas.geography.atlas.items.Relation;
+
+/**
+ * Used by the {@link MultiEdge} to hold multiple versions of identical {@link Edge}s. This is in
+ * case one of the {@link Edge}s has a parent {@link Relation} that was not contained in one of the
+ * sub-{@link Atlas}es of the containing {@link MultiAtlas}.
+ *
+ * @author lcram
```

Comment:
Wait did I write this?

Reply:
Looks like it!

Reply:
I guess I'm just confused because it looks like you added this class from scratch! Haha

Reply:
I copy pasted another class you wrote and made minimal changes to make it work for edges. So you are still the author mostly!